lib/Gateway.js

```diff
@@ -1608,6 +1609,14 @@ Gateway.prototype.discoverClimates = function (node) {
 Gateway.prototype.discoverValue = function (node, vId) {
   const valueId = node.values[vId]
 
+  // List of values
+  const ignoreDiscovery = {
+    Meter: ['previousValue', 'deltaTime']
+  }
+  if (ignoreDiscovery[valueId.commandClassName]) {
+    if (ignoreDiscovery[valueId.commandClassName].includes(valueId.property))
+      return
+  }
```

Comment:
I like this but for now there are much other cc we are ignoring, IMO this check should be done inside the CC switch-case, like `if(property === x || property === y) return`

Reply:
thought of. looking if we have second class, otherwise i will move it under. the idea was to cover all